rust/src/build.rs

```diff
@@ -0,0 +1,45 @@
+// builds.rs for Suricata
+//
+// Currently this build.rs only uses bindgen to build some Rust
+// bindings to the Suricata C code.
+//
+// For more info on Rust and the build.rs file, see:
+//    https://doc.rust-lang.org/cargo/reference/build-scripts.html
+fn main() {
```

Comment:
Do we want build-time code execution ? 😢

Reply:
We already have it for the derive macros.   The Lua crate copies in headers at build time as well.

Reply:
Many crates do that already I know 😢